ParisBNF/abb/BNFabb30.xml

```diff
@@ -0,0 +1,720 @@
+                                 </item>
+
+                                  <item xml:id="a2">
+                                    <locus from="1r" to="2r"/>
+                                     <desc type="GuestText" corresp="LIT2872RepCh153">A
+                                       malkǝʾ-hymn for the Four Living Creatures.</desc>
+                                      <q xml:lang="gez"></q>
+                                     </item>
+
+                                     <item xml:id="a3">
+                                       <locus target="#3r"/>
+                                        <desc type="OwnershipNote"/>
+                                          <q xml:lang="gez">ይትባርክ᎓ እግዚአብሔር፡ አምላከ᎓ እስራኤል᎓ ዝመጽሐፍ᎓
+                                            ለአባ᎓ አሕዋን᎓ no. 30</q>
+                                           <note>The note is written in a crude hand, in the upper margin,
+                                             in two column. It mentions an owner,
+                                             <persName ref="PRS12914Ahwan" role="owner">the Falāšā manʾabbā ʾAḥwān</persName> from whom
+                                             <persName ref="PRS1071dAbbadi">Antoine d'Abbadie</persName>
+                                             bought the manuscript. The number 30 could have been added later.</note>
+                                         </item>
+
+                                         <item xml:id="a4">
+                                           <locus from="#202va" to="203ra"/>
+                                            <desc type="GuestText"/>
+                                              <q xml:lang="gez"></q>
+                                              <note>The note appears as a continuation of <ref type="mspart" corresp="ms_i8"/>,
+                                                added at a later time, in a less professional hand (the <foreign>tamallas</foreign>-sign
+                                                sing inticates that it goes on on the next page).
+                                                It tells the chronology starting from King Yosṭos
+                                              <date>(r. 1711-16)</date> to King Takla Hāymānot, and stops on the
+                                              evens from ca. <date>1772</date>.
+                                              .</note>
+                                              </item>
+
+                                              <item xml:id="a5">
+                                                <locus target="#202ra"/>
+                                                <desc type="DonationNote"/>
+                                                <q xml:lang="gez">ዝመጽሐፍ᎓ ዝ<persName ref="PRS12916MazraetaKrestos">መዝራዕተ᎓ ክርስቶስ᎓</persName> ወብእሲቱ᎓ ምልእተ᎓ ጸጋ᎓ ወውሉዱ᎓
+                                                  ፍስሐ᎓ ክርስቶስ᎓ ሐዋርያ᎓ ክርስቶስ᎓ ስብስትያኖስ᎓ አምደ᎓ ሚካኤል᎓ ዘአጥረዮ᎓ በንዋዩ᎓ ከመ᎓ ይኩን᎓ ለመድኀኒተ᎓ ነፍሱ᎓ ዘሰረቆ᎓
```

Comment:
I think that the family members deserve IDs :)

Reply:
Rather, I will list them in the record for Mazraeta Krestos.